### DISCUSSION TITLE
PicoTorrent 0.25

### DISCUSSION THREAD

**Discussion by vktr** [picotorrent/picotorrent#1172] [General]
With the warm spring weather comes a fresh new release of PicoTorrent straight from the app bakery here in Stockholm 🌞 It's been a while since the last release, so here's a recap of what's new since last time!

🐛 **The bug fixes**
* #1126 - Use a static UUID for single instance checking to better avoid multiple instances - by @SadE54
* #1138 - Fix assigning labels when multiple torrents are selected
* #1141 - The delete key removes the torrent when tracker view has focus

💟 **The feature additions**
* #1124 - When adding files from the command line, users can now pass the save path to use as well - by @SadE54
* #1136 - Support sorting by torrent status
* #1137 - Torrent ratio is calculated using `all_time_upload / torrent_size`
* #1164 - Support label filtering in PQL

❤️ **The love** (updated translations)
* 🇮🇩 Indonesian by @zmni
* 🇮🇹 Italian by @bovirus
* 🇹🇷 Turkish by @xzork, @hipofiz
* 🇸🇦 Arabic (Saudi Arabia) by @gitntel
* 🇺🇦 Ukranian by @molny, @mxve 
* 🇬🇪 Georgian by @arabuli
* 🇻🇳 Vietnamese by @Doraemon-kun
* 🇵🇱 Polish by @luki446
* 🇷🇸 Serbian by @laikexpert, @artyom2034
* 🇯🇵 Japanese by @xE0F9
* 🇬🇧 English by @comradekingu
* 🇳🇴 Norwegian (Bokmål) by @comradekingu
* 🇧🇬 Bulgarian by @StoyanDimitrov
* 🇫🇷 French by @TechnoGreg
* 🇷🇺 Russian by @AHOHNMYC
* 🇱🇰 Sinhalese by @Aikatsui
* 🇪🇸 Spanish by @DerianAndre
* 🇧🇷 Portuguese (Brazil) by @nero-0, @mmagian
* 🇩🇪 German by @mxve
* 🇭🇺 Hungarian by @mxve
* 🇱🇹 Lithuanian by @mxve
* 🇱🇻 Latvian by @mxve
* 🇸🇪 Swedish by @mxve
* 🇨🇿 Czech by @DavidJares
* 🇳🇱 Dutch by @Stephan-P
* 🇸🇰 Slovak by @xzpyth
* 🇨🇳 Chinese (Simplified) by @mayli, @KaMingChung
* 🇭🇷 Croatian by @migwynkriid

💵 As always - [I'm on GitHub sponsors](https://github.com/sponsors/vktr) if you want to support development.
🗨️ If you just want to hang out, head on over to [the PicoTorrent Discord server](https://discord.gg/tV3dFrv)

<hr /><em>This discussion was created from the release <a href='https://github.com/picotorrent/picotorrent/releases/tag/v0.25.0'>PicoTorrent 0.25</a>.</em>

**Comment by comradekingu** [comment]
Could you use my name, and link to the account if you want. (On the assumption that is shipped anywhere.)
It is also an opportunity to link to https://translate.picotorrent.org/projects/picotorrent/

**Reply by vktr** [reply]
Do you mean in the release notes? There's no author mentions in the shipped application, just a link to GitHub 😃
